openreview/conference/builder.py

```diff
@@ -1064,6 +1063,15 @@ def setup_final_deadline_stage(self, force=False, hide_fields=[]):
             hide_fields=hide_fields
         )
 
+        if not self.submission_stage.double_blind and pc_revision_stage:
+            self.submission_revision_stage = SubmissionRevisionStage(
+                name='Revision',
+                start_date=None if force else self.submission_stage.due_date,
+                due_date=None,
+                multiReply=True
```

Comment:
could you add a property to the SubmissionRevisionStage class to represent the invitees? it could be called: `program_chairs_only=False`, the default value indicates that program chairs and authors can make edits.